zrml/prediction-markets/src/benchmarks.rs

```diff
@@ -371,35 +352,37 @@ benchmarks! {
     }: { Pallet::<T>::handle_expired_advised_market(&market_id, market)? }
 
     internal_resolve_categorical_reported {
-        // a = total accounts
-        let a in 0..10;
-        // b = num. accounts with assets
-        let b in 0..10;
-        // c = num. asset types
-        let c in (T::MinCategories::get().into())..T::MaxCategories::get().into();
-
-        let c_u16 = c.saturated_into();
-        let (_, market_id) = setup_resolve_common_categorical_after_dispute::<T>(a, b, c_u16)?;
+        let total_accounts = 1000;
+        let accounts_with_assets = 750;
+        let categories : u16 = T::MaxCategories::get().saturated_into();
+        let (_, market_id) =
+            setup_resolve_common_categorical_after_dispute::<T>(
+                total_accounts,
+                accounts_with_assets,
+                categories
+            )?;
     }: {
         let market = T::MarketCommons::market(&market_id)?;
         let disputes = crate::Disputes::<T>::get(market_id);
         T::SimpleDisputes::on_resolution(&disputes, &market_id, &market)?
     }
 
     internal_resolve_categorical_disputed {
-        // a = total accounts
-        let a in 0..10;
-        // b = num. accounts with assets
-        let b in 0..10;
-        // c = num. asset types
-        let c in (T::MinCategories::get().into())..T::MaxCategories::get().into();
         // d = num. disputes
         let d in 0..T::MaxDisputes::get();
 
-        let c_u16 = c.saturated_into();
-        let (caller, market_id) = setup_resolve_common_categorical_after_dispute::<T>(a, b, c_u16)?;
-
-        for i in 0..c.min(d) {
+        let total_accounts = 1000;
+        let accounts_with_assets = 750;
+        let categories = T::MaxCategories::get();
+        let (caller, market_id) =
+            setup_resolve_common_categorical_after_dispute::<T>(
+                total_accounts,
+                accounts_with_assets,
+                categories,
+            )?;
```

Comment:
```suggestion
        let (caller, market_id) = setup_resolve_common_categorical_after_dispute::<T>(0, 0, categories)?;
```

Reply:
I think we can remove the number of accounts from the helper function to avoid having to use `0, 0` as parameters always.

Reply:
I prefer to keep those parameters, as if we have better solution like using pallet-assets or anything else we can use it.